src/pages/index.tsx

```diff
@@ -2,10 +2,25 @@ import { useEffect } from "react";
 import { useRouter } from "next/router";
 import { defaultViewport } from "@/config/defaults";
 
+/**
+ * The main page component that redirects visitors to a default or saved location.
+ *
+ * This component uses the Next.js router to redirect users based on their saved
+ * latitude, longitude, and zoom level in the local storage. If no saved location
+ * is found, it redirects to a default location defined in the configuration.
+ * 
+ * @group Pages
+ */
 const Home = () => {
   const router = useRouter();
 
-  // Redirect to default or saved location
+  /**
+   * useEffect hook to handle the redirection logic when the component is mounted.
+   *
+   * It retrieves the saved latitude, longitude, and zoom level from the local storage.
+   * If found, it redirects to the saved location. Otherwise, it redirects to the
+   * default location specified in the `defaultViewport` configuration.
+   */
```

Comment:
This is a personal preference but I think this comment and some of the others are useful but a bit on the verbose side

Reply:
I'll revert this one.